build.gradle

```diff
@@ -26,10 +26,11 @@ repositories {
 }
 
 dependencies {
-  implementation 'org.apache.tuweni:tuweni-bytes'
-  implementation 'org.apache.tuweni:tuweni-crypto'
-  implementation 'org.apache.tuweni:tuweni-rlp'
-  implementation 'org.apache.tuweni:tuweni-units'
+  implementation 'io.vertx:vertx-core'
```

Comment:
What is the need of vertx-core dependency?

Reply:
It's a dependency for tuweni. Building locally, it would only compile if I added the dependency. For some reason tuweni wasn't bringing in that dependency automatically.

Reply:
sure, sounds good, we bring in that dependency in Teku anyways